awswrangler/redshift.py

```diff
@@ -1057,6 +1058,12 @@ def unload_to_files(
         partition_str: str = f"\nPARTITION BY ({','.join(partition_cols)})" if partition_cols else ""
         manifest_str: str = "\nmanifest" if manifest is True else ""
         region_str: str = f"\nREGION AS '{region}'" if region is not None else ""
+        if not max_file_size and engine.get() == EngineEnum.RAY:
```

Comment:
Should we do the same in Athena too? In fact I don't think we have added max_file_size in athena unload which we said we would

Reply:
Athena's [UNLOAD](https://docs.aws.amazon.com/athena/latest/ug/unload.html) doesn't seem to have this setting unfortunately